timestampable.go

```diff
@@ -0,0 +1,32 @@
+package kallax
+
+import "time"
+
+// Timestamp modelates an object that knows about when was created and updated
+type Timestamps struct {
+	// CreatedAt is the time where the object was created
+	CreatedAt time.Time
+	// UpdatedAt is the time where the object was updated
+	UpdatedAt time.Time
+	now       func() time.Time
+}
+
+// Timestamp updates the UpdatedAt and creates a new CreatedAt if it does not exist
+func (t *Timestamps) Timestamp() {
+	if t.now == nil {
+		t.now = time.Now
+	}
```

Comment:
@Serabe if the Model has a "now generator", it will use it;
if it has not, it will use the default one: "time.Now"